lib/array_intersection.rb

```diff
@@ -1,3 +1,45 @@
+## Array Intersection
+
+# Design and implement a method that takes two integer arrays with unique values and returns their intersection in a new array.
+
+# For example:
+
+# ```
+# intersection([2, 3, 4], [4, 5, 6]) => [4]
+# intersection([50, 43, 25, 72], [25, 36, 43, 50, 80]) => [50, 25, 43]
+# intersection([9, 30, 42], [56, 34, 90, 32]) => []
+# ```
+
+
+
 def intersection(list1, list2)
```

Comment:
This works, but it's O(n * m) in time complexity.  Using a hash you can get this to O(n + m) time complexity which is much better.  

I encourage you to think about how to do this.